### PR TITLE
Invoke enhance

### PR DESCRIPTION
**Pull request by mariopil**
Following improvements have been made:
- contracts code is dynamically loaded from github repo
- list of contracts is also loaded from the repo (however this does not affect the contracts dropdown)
- monaco editor has been used for the code editor
- each contracts public method has now `Invoke` button - when clicked a dialog window with example invoke command is displayed
- default values are used for contract method parameters - `1` for numeric values, a `param_name_address` for `param_name` variable of type `Address`
- inputs for init method params have been removed
- `author` and `license` inputs have been moved above buttons

### CODE REVIEW

**Review by eloylp — APPROVED** [review]
Awesome. It was easy to review. I really appreciated the PR description. Thanks for doing that.

The monaco editor looks awesome :rocket:  . Well all the ehnahcements in truth :smile: 

There is only one nitpick. It was hard to find the `invoke` button on each contract code public function. But maybe its just my resolution plus i am getting old:

image

I think we can move forward and check this later. Thanks for the great work !

File: app/src/contractsources/contractsreporeader.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import fetch from 'node-fetch'
+
+export class ContractsRepoReader {
+  private readonly ghUrl =
+    'https://api.github.com/repos/eigerco/nebula/contents'
```
[the remark this answers is not in the document]
Comment (mariopil) [reply app/src/contractsources/contractsreporeader.ts:5]: Yeah the button is a bit small. I'll check if there is a way to make it bigger.

**Comment by eloylp** [comment]
Great ! thanks @mariopil . Should we merge this ?

**Comment by mariopil** [comment]
Yeah, we can merge it :)

**Pull request merged**